client/js/widgets/widget.js

```diff
@@ -1281,6 +1282,52 @@ export class Widget extends StateManaged {
         }
       }
 
+      if(a.func == 'SCORE') {
+        setDefaults(a, { mode: 'set', property: 'score', seats: null, round: null, value: null});
+        if([ 'set', 'inc', 'dec' ].indexOf(a.mode) == -1) {
+          problems.push(`Warning: Mode ${a.mode} interpreted as set.`);
+          a.mode = 'set'
+        }
+        
+        if(a.value === null)
+          a.value = a.mode=='set' ? 0 : 1;
+        if(isNaN(parseFloat(a.value))) {
+          problems.push(`value ${a.value} must be a number, assuming 0.`);
+          a.value = 0;
+        }
+        a.value = parseFloat(a.value);
+
+        let round = a.round;
+        if(round != null && (isNaN(parseInt(round)) || round < 1)) {
```

Comment:
Are you sure that `round != null` shouldn't be `round !== null`? I'm pretty sure that nothing but `null` is `== null` in modern Javascript implementations but it could be anything...

Reply:
Fixed.